UnityProject/Assets/Scripts/Objects/Other/SecBarrier.cs

```diff
@@ -0,0 +1,68 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace Objects.Security
+{
+	[RequireComponent(typeof(AccessRestrictions))]
+	[RequireComponent(typeof(Construction.WrenchSecurable))]
+	public class SecBarrier : MonoBehaviour, IPredictedCheckedInteractable<HandApply>
```

Comment:
make it jsut a ``CheckedInteractable`` if you're not going to implement logic for the prediction.

Reply:
Done!